ExeterTeam25/group-software-project
# Responsive css

To test:
- Inspect element and press Ctrl+Shift+m
- Go around the website trying the different screen sizes at the top and make sure everything works
- Not finished yet, looking on where is best to improve

Known issues:
- div boxes and images squash at times
- Gap on right hand size through the pages
- Login page still doesn't fit whole screen

What's been added:

- Toggle the sidebar
- Sidebar automatically hidden on smaller screens
- Map and challenges adjust to screen size
